pages/common/xml.md

```diff
@@ -0,0 +1,29 @@
+# xml
+
+> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
+> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, or `validate`.
```

Comment:
Perhaps this will read better?
```suggestion
> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, and `validate`.
```

Reply:
I disagree. "Or" is appropriate for alternatives, which these are.

❯ grep ", or" *.md
alex.md:> It helps you find gender **favouring, polarising, race related, religion inconsiderate, or other** unequal phrasing in text.
circo.md:- Render the output in **`ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot`** format:
ctest.md:- Run a single test based on its name, or filter on a regular expression:
docker-inspect.md:- Display information about a **container, image, or volume** using a name or ID:
dot.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
dvc-config.md:> These configurations can be on **project, local, global, or system** level.
dvc-config.md:- Set a **local, global, or system** level config value:
fdp.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
fzf.md:- Start fzf on entries that start with core and end with either **go, rb, or py:**
gifsicle.md:- You can also edit animations by **replacing, deleting, or inserting** frames:
git-abort.md:> Abort an ongoing **rebase, merge, or cherry-pick.**
git-abort.md:- Abort a Git rebase, merge, or cherry-pick:
git-check-attr.md:> For every pathname, list if each attribute is unspecified, set, or unset as a gitattribute on that pathname.
git-ls-remote.md:> If no name or URL are given, then the configured upstream branch will be used, or remote origin if the former is not configured.
git-send-email.md:> Patches can be specified as **files, directions, or a revision list**.
grep.md:- Print 3 lines of context around, before, or after each match:
httpry.md:> It can be run in real-time displaying the traffic as it is parsed, or as a daemon process that logs to an output file.
kind.md:- Get details about clusters, nodes, or the kubeconfig:
lmms.md:> Render a `.mmp` or `.mmpz` project file, dump a `.mmpz` as XML, or start the GUI.
magick.md:> Create, edit, compose, or convert bitmap images.
mmdc.md:> A mermaid definition file is taken as input and a SVG, PNG, or PDF file is generated as output.
mmdc.md:- Specify the background color of the chart (e.g. `lime`, `"#D8064F"`, or `transparent`):
mongoimport.md:> Imports content from a JSON, CSV, or TSV file into a MongoDB database.
neato.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
nl.md:> A utility for numbering lines, either from a file, or from standard input.
osage.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
pamixer.md:- Use the allow boost option to increase, decrease, or set the volume above 100%:
patchwork.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
rails-routes.md:- List routes partially matching URL helper method name, HTTP verb, or URL path:
rbash.md:> Does not permit changing the working directory, redirecting command output, or modifying environment variables, among other things.
rsync.md:> Can transfer single files, or multiple files matching a pattern.
sfdp.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
sonar-scanner.md:> SonarScanner is a generic scanner for SonarQube projects that do not use build tools such as Maven, Gradle, or Ant.
streamlink.md:- Skip to the beginning of a live stream, or as far back as possible:
tput.md:- Show number of columns, lines, or colors:
truncate.md:- Set a size of 10 GB to an existing file, or create a new file with the specified size:
twopi.md:- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
virt-sysprep.md:> Reset, unconfigure, or customize a virtual machine image.
wc.md:> Count lines, words, or bytes.
where.md:> It could be an executable in the PATH environment variable, an alias, or a shell builtin.
xxd.md:> Create a hexadecimal representation (hexdump) from a binary file, or vice-versa.

Reply:
> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
> This command also has documentation about its subcommands, e.g. `xml validate`.
> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, or `validate`.
> More information: <http://xmlstar.sourceforge.net/docs.php>.